pkg/controllers/dashboard/systemcharts/controller.go

```diff
@@ -142,13 +154,20 @@ func (h *handler) getChartsToInstall() []*chart.Definition {
 				}
 				// add priority class value.
 				if priorityClassName, err := h.chartsConfig.GetPriorityClassName(); err != nil {
```

Comment:
This follows along with the discussion above, but I think that this should probably be removed in favor of a 1-stop call to get the entire webhook values rather than getting just the priority class and merging with the larger map.

Reply:
We need to keep the current behavior because this value comes from a specific Helm value when installing Rancher and applies to multiple charts. If a user wants to change the priority class for only the Webhook, then they should leverage the new flow of `cluster.spec.chartValues`